### PR TITLE
PIM-8433: Save loading messages in the database and display them randomly on the main loading screen.

### PR DESCRIPTION
**Pull request by pchasle**
**Description (for Contributor and Core Developer)**

Two things:
- Save the _loading messages_ in the database with `oro_config`. Load the defaults from the file `loading_messages.txt` if the config doesn't exist yet. And it doesn't write in the file anymore (so it's SaaS friendly).
- Display a random loading message from the config on the main loading screen.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | OK
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | OK
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php [hunk 1]
```diff
@@ -59,16 +59,18 @@ public function getAction()
...
+
+            if ($option['name'] === 'loading_messages' && $value === null) {
+                $value = file_get_contents($this->getMessagesFilePath());
+            }
 
             $data[$viewKey] = [
-                'value'                  => $value,
-                'scope'                  => 'app',
-                'use_parent_scope_value' => false
+                'value' => $value,
```
Comment (ahocquard) [inline src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php]: don't change things if you don't touch it please

File: src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php [hunk 2]
```diff
@@ -59,16 +59,18 @@ public function getAction()
         $data = [];
 
         foreach ($this->options as $option) {
-            $viewKey = $option['section'] . ConfigManager::SECTION_VIEW_SEPARATOR . $option['name'];
-            $modelKey = $option['section'] . ConfigManager::SECTION_MODEL_SEPARATOR . $option['name'];
-            $value    = $option['name'] === 'loading_messages' ?
-                        file_get_contents($this->getMessagesFilePath()) :
-                        $this->configManager->get($modelKey);
+            $viewKey = $option['section'].ConfigManager::SECTION_VIEW_SEPARATOR.$option['name'];
```
Comment (ahocquard) [inline src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php]: what is the change except style?

File: src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php [hunk 3]
```diff
@@ -59,16 +59,18 @@ public function getAction()
...
+            $modelKey = $option['section'].ConfigManager::SECTION_MODEL_SEPARATOR.$option['name'];
+            $value = $this->configManager->get($modelKey);
+
+            if ($option['name'] === 'loading_messages' && $value === null) {
+                $value = file_get_contents($this->getMessagesFilePath());
```
Comment (ahocquard) [inline src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php]: when does it happen? As you removed lines about saving in the file in the postAction

File: src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php (hunk 2 above)
Comment (pchasle) [reply src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php]: There is none, the formatter got in the way. Sorry about that.

File: src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php (hunk 3 above)
Comment (pchasle) [reply src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php]: The default value is loaded from the file if the users never saved the settings and/or if there is no `loading_messages` key in the settings.

Comment (pchasle) [reply src/Oro/Bundle/ConfigBundle/Controller/Rest/ConfigurationController.php]: The save is automatically handled by the oro config `configManager->save`

**Pull request merged**